lib/management/push.js

```diff
@@ -86,13 +84,17 @@ function startWSManagementClient(pushEndpoint, instanceId, token) {
         }
     }
 
+    overlayMessageListener = newOverlayCallback;
+
     const socketsByChannelId = [];
     const headers = {
         'x-instance-authentication-token': token,
     };
     const agent = createWSAgent(pushEndpoint, process.env, logger);
 
-    const url = `${pushEndpoint}/${instanceId}/ws`;
+    const url = process.env.PUSH_ENDPOINT ||
```

Comment:
`${pushEndpoint}/${instanceId}/ws` should be used here as it has been correctly computed, and connecting the websocket client to `process.env.PUSH_ENDPOINT` directly will not work (it's missing paths).

Reply:
done